src/main/java/org/terasology/namegenerator/creature/CreatureAssetTheme.java

```diff
@@ -16,13 +16,13 @@
 
 package org.terasology.namegenerator.creature;
 
-import java.util.Collections;
-import java.util.List;
-
 import org.terasology.asset.Assets;
 import org.terasology.entitySystem.prefab.Prefab;
 import org.terasology.namegenerator.data.NameGeneratorComponent;
 
+import java.util.Collections;
```

Comment:
The default IDE sorting on reformat imports seems to have been working well for long now, haven't noticed any import juggling. Can't remember the last time I tried to move an import statement manually.